### PR TITLE
Sidebar order - now 2 folders use number prefix

### PR DESCRIPTION
**Pull request by mi-hol**

### CODE REVIEW

**Comment by mi-hol** [comment]
changes continue to address https://github.com/FOME-Tech/wiki/issues/102

**Review by nmschulte — APPROVED** [review]

**Comment by nmschulte** [comment]
image

These changes look good, but wondering why the titles change for the pages including "-" in the name.

File: docs/03-Intro-Start-Here/_category_.json
Comment (nmschulte) [inline docs/03-Intro-Start-Here/_category_.json:1]: I guess we should retain these files that set the `label`, so the titles retain their desired formatting?

**Comment by mi-hol** [comment]
> he titles change for the pages including "-" in the name.

thanks for catching my oversight, just need to remove the "position" setting when present

**Pull request merged**